.github/workflows/code-analysis.yml

```diff
@@ -0,0 +1,50 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  php-stan:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.4']
+    steps:
+     - name: Install PHP 7.4
+       uses: shivammathur/setup-php@v2
+       with:
+         php-version: ${{ matrix.php-versions }}
+         extensions: intl #optional
+         ini-values: "post_max_size=256M" #optional
+         
+     - name: Install prestashop                              
+       run: |
+         wget https://download.prestashop.com/download/releases/prestashop_1.7.7.0.zip
+         unzip prestashop_1.7.7.0.zip
+         unzip prestashop.zip -d prestashop
+         rm prestashop.zip prestashop_1.7.7.0.zip
+     - run: mkdir prestashop/modules/altapay
+     - uses: actions/checkout@v2 
+       with:
+          path: prestashop/modules/altapay
+     - name: Install SDK                             
+       run: |
+         git clone https://github.com/AltaPay/sdk-php.git
+         cp -r sdk-php/lib/* prestashop/modules/altapay/lib/altapay/altapay-php-sdk/lib/
+         rm -rf sdk-php
```

Comment:
why no use composer for this?

Reply:
on packagist sdk-php is available, but it's not updated and neither sync with github repo.